spec/yaml/swiftnav/sbp/imu.yaml

```diff
@@ -86,6 +86,8 @@ definitions:
                   values:
                       - 0: Bosch BMI160
                       - 1: ST Microelectronics ASM330LLH
+                      - 2: Reserved
```

Comment:
This value is currently used for Android but I'm not sure we need to make it public - @reimerix ?